translations.json

```diff
@@ -165,22 +167,25 @@
         },
         "label3.Text": {
             "en": "Status:",
-            "fr-FR": "Etat",
+            "fr-FR": "Etat :",
             "ja-JP": "ステータス :",
             "ko-KR": "상태:",
             "zh-CN": "状态："
         },
         "label4.Text": {
             "en": "Overlay:",
+            "fr-FR": "Overlay :',
```

Comment:
The quote at the end should be `"`, not `'`.

Reply:
Yep !